<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FVMClass</class>
 <widget class="QMainWindow" name="FVMClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="RenderWidget" name="glWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadConfig"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStep"/>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadConfig"/>
   <addaction name="actionStep"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionTest"/>
   <addaction name="actionImportNodes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadConfig">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/load.png</normaloff>:/icon/icon/load.png</iconset>
   </property>
   <property name="text">
    <string>LoadConfig</string>
   </property>
   <property name="toolTip">
    <string>Load Model</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/do_step.png</normaloff>:/icon/icon/do_step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Simu one time step</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/run.png</normaloff>:/icon/icon/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Start Simulation</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/pause.png</normaloff>:/icon/icon/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause simulation</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/stop.png</normaloff>:/icon/icon/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop simulation</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/test.png</normaloff>:/icon/icon/test.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="toolTip">
    <string>test</string>
   </property>
  </action>
  <action name="actionImportNodes">
   <property name="icon">
    <iconset resource="fvm.qrc">
     <normaloff>:/icon/icon/import.png</normaloff>:/icon/icon/import.png</iconset>
   </property>
   <property name="text">
    <string>ImportNodes</string>
   </property>
   <property name="toolTip">
    <string>Import Nodes Positions</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>RenderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fvm.qrc"/>
 </resources>
 <connections/>
</ui>
